src/scribe_data/extract_transform/languages/Arabic/nouns/query_nouns.sparql

```diff
@@ -1,7 +1,9 @@
 # All Arabic (Q13955) nouns.
 # Enter this query at https://query.wikidata.org/.
 
-SELECT DISTINCT ?lexeme ?noun WHERE {
```

Comment:
@andrewtavis just one thing, here I am suppose to make change, isnt it?

Reply:
Yes, that'd be great, @mhmohona :)

Reply:
So the changes I made, are they sufficient?

Reply:
This will be find, @mhmohona :) I'll go through and do the review and add in the line for you 😊